src/lib/salad/hll.h

```diff
@@ -0,0 +1,158 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#pragma once
+
+#include <stdint.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+#include "lib/salad/hll_empirical.h"
+
+/**
+ * HyperLogLog supports sparse and dense represenations.
```

Comment:
It'd be nice to have a link to (or at least the title of) the article you used somewhere in the code.

Reply:
Done. See the top comment in `hll.c`